File: tabels.py
#Python Program to Print all Numbers in a Range Divisible by a Given Number

lower = int(input("enter the lower limit"))
upper = int(input("enter the upper limit"))


j = int(input("enter the divisible value"))

for i in range(lower,upper+1):
	k = i//j
	if(i%j == 0):
		print(j,k,sep='*',end='=')
		print(i)

#for i in range(1,upper+1):
#	s= j * i
#	print(j,i,sep='*',end='=')
#	print(s)
	

File: tabels.c
#include<stdio.h>

void main()
{
	int i,lower,upper,j,product;
	printf("enter the range and divisible value: ");
	scanf("%d %d %d",&lower,&upper,&j);
	for(i=lower;i<=upper;i++)
	{
		product = i/j;
		if(i%j == 0)
			printf("%d*%d=%d \n",j,product,i);
		/*product = j * i;		
		printf("%d*%d=%d \n",j,i,product);*/
	}
}
